Sources/SwiftIDEUtils/SyntaxClassifier.swift

```diff
@@ -196,12 +196,23 @@ private struct ClassificationVisitor {
 
     for case (let index, let child?) in children.enumerated() {
 
-      let classification: (SyntaxClassification, Bool)?
+      let classification: (classification: SyntaxClassification, force: Bool)?
       if case .layout(let layout) = descriptor.node.kind.syntaxNodeType.structure {
         classification = SyntaxClassification.classify(layout[index])
       } else {
         classification = nil
       }
+
+      if let classification, classification.force == true {
+        let range = SyntaxClassifiedRange(
+          kind: classification.classification,
+          range: ByteSourceRange(offset: byteOffset, length: child.byteLength - child.leadingTriviaByteLength - child.trailingTriviaByteLength)
```

Comment:
If you are subtracting `child.leadingTriviaByteLength` from `length` you need to add it to `offset`